cmd/clusterctl/client/init.go

```diff
@@ -101,6 +106,8 @@ func (c *clusterctlClient) Init(options InitOptions) ([]Components, error) {
 	log.Info("Fetching providers")
 	firstRun := c.addDefaultProviders(clusterClient, &options)
 
+	options.allowMissingProviderCRD = true
```

Comment:
Given the name of the variable, tge right way to do it will be to set the value to `true` in `InitImages` and flip the check in `setupInstaller`.